src/Analyzers/Core/Analyzers/UseConditionalExpression/UseConditionalExpressionHelpers.cs

```diff
@@ -61,28 +58,11 @@ public static IOperation UnwrapImplicitConversion(IOperation value)
                 ? conversion.Operand
                 : value;
 
-        public static SyntaxRemoveOptions GetRemoveOptions(
```

Comment:
Split into separate files - `UseConditionalExpressionHelpers` and `UseConditionalExpressionCodeFixHelpers`

Reply:
why?

Reply:
SyntaxGenerator/SyntaxRemoveOptions is not available in CodeStyle analyzer layer, so it had to be moved to CodeStyle fixer layer and the rest of the helpers are used by the analyzer, so moved to CodeStyle analyzer layer.

Reply:
Thanks!